### PR TITLE
Addition of Azure Function App Sample

### PR DESCRIPTION
**Pull request by vnextcoder**
# Fixes # 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:
- This adds a Azure Function App Sample for integrating into SendGrid 

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-csharp?pullRequest=640) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-csharp?pullRequest=640) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-csharp?pullRequest=640) it.</sub>

**Comment by vnextcoder** [comment]
Any updates when this is going to be merged?

**Comment by thinkingserious** [comment]
Hello @vnextcoder,

I need at least one code review. Right now, it's on our backlog for a code review. 

A code review from the community could also speed up the process.

Thanks for your patience and my apologies for the delay.

With Best Regards,

Elmer

File: ExampleAzureFunctionapp/ManualTriggerFunctionCS/function.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "bindings": [
+    
+    {
+      "name": "$return",
+      "type": "sendGrid",
```
Comment (thinkingserious) [inline ExampleAzureFunctionapp/ManualTriggerFunctionCS/function.json:6]: Should this be `SendGrid`, based on the Output Binding description in the README.md?

File: ExampleAzureFunctionapp/ManualTriggerFunctionCS/function.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      "direction": "out",
+      "apiKey": "SENDGRIDEKY"
```
Comment (thinkingserious) [inline ExampleAzureFunctionapp/ManualTriggerFunctionCS/function.json:8]: typo

File: ExampleAzureFunctionapp/ManualTriggerFunctionCS/run.csx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+#r "SendGrid"
+using System;
+using SendGrid.Helpers.Mail;
+
+public static Mail Run(TraceWriter log, string input)
+{
+    Mail message = new Mail
+    {        
+        Subject = "Azure newsletter"
```
Comment (thinkingserious) [inline ExampleAzureFunctionapp/ManualTriggerFunctionCS/run.csx:9]: nit: trailing spaces

File: ExampleAzureFunctionapp/ManualTriggerFunctionCS/run.csx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+public static Mail Run(TraceWriter log, string input)
+{
+    Mail message = new Mail
+    {        
+        Subject = "Azure newsletter"          
+    };
+
+    var personalization = new Personalization();
+    personalization.AddTo(new Email("recipient@contoso.com"));
```
Comment (thinkingserious) [inline ExampleAzureFunctionapp/ManualTriggerFunctionCS/run.csx:13]: nit: trailing spaces

Lets please use the example.com domain.

File: ExampleAzureFunctionapp/ManualTriggerFunctionCS/README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+**Overview**
+
+This Library allows you to setup a function inside Azure Function App quickly with bare metal frame structure. 
+
+*Input Binding* - Manual Trigger using Azure Portal admin console.
+
+*Output Binding* - SendGrid, which is bound using the return type of the function in function.json
```
Comment (thinkingserious) [inline ExampleAzureFunctionapp/ManualTriggerFunctionCS/README.md:7]: Please add some instructions on how to set this up and what the expected output will be.

**Review by thinkingserious — COMMENTED** [review]
Thank you for the PR, we greatly appreciate it!

**Comment by thinkingserious** [comment]
Closing until PR feedback is addressed.

**Pull request closed without merging**